### DISCUSSION TITLE
MARK feature writing

### DISCUSSION THREAD

**Discussion by cjdunn** [koeberlin/Latin-Character-Sets#10] [Q&A]
@koeberlin 
You said 
`For a wider language support, a proper mark feature is recommended.`
[https://github.com/koeberlin/Latin-Character-Sets#mark](url)

Do you have any resources/examples for writing proper MARK features? For example to render the Yoruba name: Olúfẹ́mi***
so that the /edotbelow and the /acutecomb will be aligned. The MARK feature auto-generated by fontmake (via [ufo2ft](https://github.com/googlefonts/ufo2ft/blob/main/Lib/ufo2ft/featureWriters/markFeatureWriter.py)) is not doing the trick here. Any suggestions would be much appreciated, thanks! 

***note: even github, on my end, is not rendering this correctly, though it does render correctly on your Pangea site <https://fontwerk.com/en/fonts/pangea>

<img width="979" alt="Olúfẹ́mi_Pangea" src="https://user-images.githubusercontent.com/3900241/168693335-5c11d4e2-c4d9-4d20-83f5-c74000cfefba.png">

**Comment by koeberlin** [comment]
Looks like the system San Franciscos don’t have a proper `mark` feature for Latin (at least on my system):
<img width="558" alt="Bildschirmfoto 2022-05-17 um 10 15 20" src="https://user-images.githubusercontent.com/93979655/168908387-10657ab0-fa7a-487a-8ba3-802889442e8e.png">
Pangea is created with Glyphs, which does a pretty good job converting the anchors into marks. I’ll have a look how to achieve it in other workflows!

**Reply by koeberlin** [reply]
UFO + fontmake, as shown by @arrowtype ☞ https://www.youtube.com/watch?v=mm7GJ39ojXI

**Reply by arrowtype** [reply]
But isn’t the font in the input field system SF? There, it seems like the mark feature is working...

<img width="553" alt="image" src="https://user-images.githubusercontent.com/45946693/195118067-19f93bef-3b29-4eeb-ba6f-8f33aa4980bb.png">

**Reply by koeberlin** [reply]
Good catch! But seems just misaligned in another way
<img width="188" alt="Bildschirmfoto 2022-10-11 um 18 12 11" src="https://user-images.githubusercontent.com/93979655/195144600-547c156b-55ba-4737-801a-e4636660cc25.png">

**Reply by arrowtype** [reply]
Slightly, yes. But it's hard to tell if that is an anchor misplacement, a different opinion, or mark feature issue...
